CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+# 4.18.0 / 2023-07-04
+ * [FEATURE] Update recipes to support amazonlinux 2023 installation [#896]
+ * [FEATURE] Add agent configuration value to config template [#888]
+ * [FEATURE] Actually enforce `apt` and `yum` versions in the `metadata.rb` file [#874]
+ * [FEATURE] support weblogic integration [#891]
+ * [FEATURE] Add new APT and RPM signing keys [#890]
+ * [BUGFIX] Fix dead link in http_check.rb recipe [#887]
```

Comment:
Shouldn't we add these lines' author?

Reply:
Also, for this, you'll have to add yourself at the end of the file (contributor's link to their profile)

Reply:
Let's do it now and we'll decide if we keep this forever